apiserver/errors/errors.go

```diff
@@ -111,6 +111,43 @@ func IsUpgradeSeriesValidationError(err error) bool {
 	return ok
 }
 
+// errIncompatibleSeries is a standard error to indicate that the series
+// requested is not compatible with the charm of the application.
+type errIncompatibleSeries struct {
```

Comment:
The one in state is soon to go, for now, duplicate it to apiserver, so that we can use it for prepare.